Add case study: CW20 (#44)
softwaresaved/event-organisation-guide: docs/eog/case-studies-cw20-infrastructure.rst

.. _CW20-Infrastructure: 

CW20 Infrastructure
====================

We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 

> We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 

Reviewer: ```suggestion
We emailed registrants through Eventbrite to notify them of the decisions that we were making and thereby keep the informed. 
```

> Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 

Reviewer: ```suggestion
Because of the unconference nature of the event, we even asked for participant input to our contingency investigation; surveying them to see if they preferred the event to stay as an in person event, move to online or be cancelled. 
```

As we worked to reformat the event and move it online, we gave registrants time to cancel and collect full refunds. 
We also reduced the cost of the event (from £250 to £50) and issued partial refunds through Eventbrite to participants who wanted to stay and participate in the virtual event. 

> We took the partial refund route in order to maintain continuity with our registrants, but another option could have been to outright cancel the Eventbrite event, issue full refunds and start from scratch. 

Reviewer: Might be worth saying - why the other option might have been worth considering - I think there was a different fee impact if I remember.

We also provided `guidance <https://software.ac.uk/news/collaborations-workshop-2020-cw20-move-online>`_ and documentation on reclaiming costs to help participants who had already booked their travel to Belfast.

For the in-person event, we had planned to use additional tools such as `Slack <https://slack.com>`_, `Slido <https://www.sli.do/>`_ and `uCONFLY <http://uconfly.org/>`_, however for the online event we felt that it was important to minimise the number of platforms that participants had to register with and keep track of. 
We narrowed our needs down to platforms for video conferencing, a chat system for sharing information and community engagement, and a tool for collaborative note-taking to keep everyone synchronised. 

Platforms
----------

Video conferencing
^^^^^^^^^^^^^^^^^^^

> `Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (namely breakout rooms and easy recording of the event). 

Reviewer: ```suggestion
`Zoom <https://zoom.us/>`_ was very quickly identified as the video conferencing platform that we would use because we had the most experience with it and it had the functionality we needed (scaling and stability with many video based participants, breakout rooms and easy recording of the event). 
```

Although Zoom has chat functionality, it can be distracting for hosts/speakers and difficult to manage when there are many participants using it. 
We decided to disable the Zoom chat (except for the case where participants could communicate directly with the meeting Host) and direct engagement and communication to the CW20 Slack workspace, as this would allow participants to connect even when the Zoom room is closed. 
A communication channel that persists outside the meeting is helpful for maintaining access to resources shared, referencing discussions and facilitating community engagement. 
We created practical channels for event organisation and communication, such as a ``#help-desk`` channel for participants to direct their technical issues to and private backchannels for the organisers, helpers, and Code of Conduct committee. 
Participants were able to create channels for their Discussion groups and Hackday teams. 
Slack also allowed for a more human and informal engagement through the creation of channels such as ``#pets-at-cw20``, ``#kids-at-cw20`` and ``#outside-my-window-cw20`` for sharing photos of our pets, children and working from home environment during lockdown.

> Collaborative documenting

Reviewer: Is it worth mentioning the limit we reached with Google Docs - i.e. 100 connections before you had to make it public to support that many connections - and we could not do that as it would open the event to disruption. And perhaps a caveat that at the time of writing HackMD although requiring the knowledge of some syntax can support over 240+ connections.

^^^^^^^^^^^^^^^^^^^^^^^^^^

The final resource that we needed to decide on was what platform to use for collaborative note-taking - a method for participant interaction and engagement that we learned from communities such as `Mozilla Open Leaders <https://foundation.mozilla.org/en/initiatives/mozilla-open-leaders/>`_, `rOpenSci <https://ropensci.org/commcalls/>`_ and `NL-RSE Meetups <https://nl-rse.org/pages/meetups.html>`_. 
Google Docs was the obvious choice for CW20, as we generate a lot of Google Docs through uCONFLY for the various sessions, and it also has the lowest barrier to accessibility (no need to log in or register and you do not need to know specific syntax or markdown language such as with `HackMD <https://hackmd.io>`_). 
We created a note-taking document for each day which contained the full agenda and all relevant event information, a space for roll call, instructions and links (links to Zoom rooms for parallel sessions, Discussion group and Collaborative Ideas documents, etc.) as can be seen in this `example collaborative document <https://doi.org/10.6084/m9.figshare.12498257>`_. 

> There were spaces created within the document for people to take notes during the talks and to ask any questions; all of which could be preserved and referred back to in the future. 

Reviewer: ```suggestion

There were spaces created within the document for people to take notes during the talks and to ask any questions; all of which could be preserved and referred back to in the future. 
```

Reviewer: It's a long paragraph and I think it can be made into two at this point.

We also provided space for feedback at the end of the document on what worked well during the workshop and what didn’t so that we could adapt for the next day. 

Normally, we require CW participants to register on the SSI’s unconference resource management system uCONFLY, which allows them to request instances of Google Docs for their group sessions and vote for the best Collaborative Ideas. 
It also allows all of the outputs from CW to be centralised. 
For CW20, we used uCONFLY purely as an admin tool by creating the templates for the Discussion, Collaborative Ideas and Mini-workshop sessions, and generated all of the needed instances ahead of time ourselves. 
We then collated them via Google Sheets (see this `example spreadsheet <https://doi.org/10.6084/m9.figshare.12498278>`_) which we linked to in the day’s collaborative note-taking document/agenda. 

> Resources

Reviewer: It might be worth making the list into two sublist - one for hosts-only and one for everyone (i.e. hosts and participants) - I think it will make it clearer for those who are following along to use the case study.
